server/sandbox_remove.go

```diff
@@ -74,27 +71,10 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 		}
 	}
 
-	if err := label.ReleaseLabel(sb.processLabel); err != nil {
-		return nil, err
-	}
-
-	// unmount the shm for the pod
-	if sb.shmPath != "/dev/shm" {
-		if mounted, err := mount.Mounted(sb.shmPath); err == nil && mounted {
-			if err := syscall.Unmount(sb.shmPath, syscall.MNT_DETACH); err != nil {
-				return nil, err
-			}
-		}
-	}
-
-	if err := sb.netNsRemove(); err != nil {
-		return nil, fmt.Errorf("failed to remove networking namespace for sandbox %s: %v", sb.id, err)
-	}
-
 	s.removeContainer(podInfraContainer)
```

Comment:
That's stated here https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto#L29-L30 - I won't add it here again as if CRI changes it we need to change it here as well